docs/quickstart.ipynb

```diff
@@ -9,6 +9,12 @@
    "outputs": [],
```

Comment:
<p>See previous comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I would again rename "Recombination coefficients". Sorry, that this was unclear.

Reply:
Rename to what? At this moment it's called "Zeta data"

Reply:
It's done